[Jules]: fix the typescript build pipeline

### Task Requirements

You are an Senior Frontend Architect specializing in React and TypeScript. Your task is to resolve a series of compilation errors preventing the `meraki-ha-www` build from succeeding.

<detected_errors>
1. TS2307: Cannot find module './types/websocket' - The API Contract file is missing or unreadable.
2. TS2339: Property 'ha-...' does not exist on type 'JSX.IntrinsicElements' - TypeScript does not recognize Home Assistant's native Web Components.
3. TS6133: 'hass' is declared but its value is never read in `DeviceTable.tsx`.
</detected_errors>

<instructions>
1. Check the project requirements to prevent regressions and update them based on the fix.
2. Think step-by-step. We need to establish the global type definitions for Home Assistant elements and ensure the WebSocket contract is properly exported.
3. Action 1: Create (or fix) `src/types/websocket.ts` with the `WsCommand` enum and relevant interfaces as previously designed.
4. Action 2: Create a global declaration file `src/types/hass-elements.d.ts`. Inside, extend the `JSX.IntrinsicElements` interface to include the following tags:
   - `ha-circular-progress` (with `active` and `alt` props)
   - `ha-settings-row`
   - `ha-icon` (with `icon` and `style` props)
   - `ha-tabs` and `paper-tab`
5. Action 3: Refactor `src/components/DeviceTable.tsx` to remove the unused `hass` variable or prefix it with an underscore (`_hass`) to satisfy the linter.
6. Ensure `tsconfig.json` is configured to include the new `src/types` directory in its compilation path.
7. Always output the complete merged files and proactively apply optimizations.
8. Output the code for `src/types/websocket.ts`, `src/types/hass-elements.d.ts`, and the refactored `DeviceTable.tsx`.
</instructions>

Please ask me to provide your `tsconfig.json` and the current `DeviceTable.tsx` inside <input_code> tags to begin.

### Specific Constraints

Ignore all errors not related to the meraki_ha integration. Follow Home Assistant Sentence Case standards.

--- a/custom_components/meraki_ha/www/src/components/DeviceTable.tsx
+++ b/custom_components/meraki_ha/www/src/components/DeviceTable.tsx
@@ -8,7 +8,7 @@ interface DeviceTableProps {
 }
 
 const DeviceTable: React.FC<DeviceTableProps> = ({
-  hass,
+  hass: _hass,
   devices,
   setActiveView,
   deviceType = 'other',

